packages/applications-service-api/src/lib/blobStorage.js

```diff
@@ -0,0 +1,18 @@
+const { DefaultAzureCredential } = require('@azure/identity');
+const { BlobServiceClient } = require('@azure/storage-blob');
+
+const config = require('./config');
+
+const upload = async (buffer, mimeType, path) => {
+	const client = new BlobServiceClient(
```

Comment:
Just thinking about the unhappy path where that client can't be created (missing/incorrect config), would we get a reasonable error logged?

Reply:
I tested it out and the error is a bit vague in the scenario where the storage container URL is undefined, so will add in some error handling and logging

Reply:
Added a couple of try/catch blocks to allow us to log a more meaningful error. The exception details will get caught and logged higher up the stack so haven't logged the error itself here.